src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationLambdaInEnum.java

```diff
@@ -0,0 +1,86 @@
+ * tabWidth = 8                                                                     //indent:1 exp:1
+ * basicOffset = 4                                                                  //indent:1 exp:1
+ * braceAdjustment = 0                                                              //indent:1 exp:1
+ * caseIndent = 4                                                                   //indent:1 exp:1
+ * throwsIndent = 4                                                                 //indent:1 exp:1
+ * forceStrictCondition = false                                                     //indent:1 exp:1
+ * lineWrappingIndentation = 4                                                      //indent:1 exp:1
+ */                                                                                 //indent:1 exp:1
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+import java.util.function.Predicate; //indent:0 exp:0
+
+import java.util.List; //indent:0 exp:0
+
+public enum InputIndentationLambdaInEnum { //indent:0 exp:0
+	/** //indent:8 exp:8
+	 * Getter. //indent:9 exp:9
+	 * T get() //indent:9 exp:9
+	 */ //indent:9 exp:9
+	GET(-1, false, 1, m -> m.isEmpty() //indent:8 exp:8
+		&& (m.getClass().getSimpleName().startsWith("get") || m.getClass().getSimpleName().startsWith("is"))), //indent:16 exp:16
+	/** //indent:8 exp:8
+	 * Setter //indent:9 exp:9
+	 * void set(T) //indent:9 exp:9
+	 */ //indent:9 exp:9
+	SET(0, false, 1, m -> m.size() == 1 && m.getClass().getSimpleName().startsWith("set")), //indent:8 exp:8
+
+	// both of the following levels of indentation are acceptable //indent:8 exp:8
+	/** //indent:8 exp:8
+	 * void add_w(T) //indent:9 exp:9
+	 */ //indent:9 exp:9
+	ADD_W(0, true, 10, m -> { //indent:8 exp:8
+	if (m.size() == 1) { //indent:8 exp:8
+	    if (m.getClass().getSimpleName().startsWith("add") || m.getClass().getSimpleName().startsWith("insert")) { //indent:12 exp:12
+	        return m.getClass().getSimpleName().endsWith("AtTop") || m.getClass().getSimpleName().endsWith("Begin"); //indent:16 exp:16
+	    } //indent:12 exp:12
+	} //indent:8 exp:8
+	return false; //indent:8 exp:8
+	}), //indent:8 exp:8
+	/** //indent:8 exp:8
+	 * void add_x(T) //indent:9 exp:9
+	 */ //indent:9 exp:9
+	ADD_X(0, true, 10, m -> { //indent:8 exp:8
+	    if (m.size() == 1) { //indent:12 exp:12
+	        if (m.getClass().getSimpleName().startsWith("add") || m.getClass().getSimpleName().startsWith("insert")) { //indent:16 exp:16
+	            return m.getClass().getSimpleName().endsWith("AtTop") || m.getClass().getSimpleName().endsWith("Begin"); //indent:20 exp:20
+	        } //indent:16 exp:16
+	    } //indent:12 exp:12
+	    return false; //indent:12 exp:12
+	}), //indent:8 exp:8
+	/** //indent:8 exp:8
+	 * void add_y(T) //indent:9 exp:9
+	 */ //indent:9 exp:9
+	ADD_Y(0, true, 1, m -> {  //indent:8 exp:8
+			if (m.size() == 1) { //indent:24 exp:8,12 warn
```

Comment:
Since `forceStrictCondition` is false, why should this be a violation?

Reply:
``` cat config.xml test.java                 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name = "Checker">
    <property name="charset" value="UTF-8"/>

    <property name="severity" value="warning"/>

    <property name="fileExtensions" value="java, properties, xml"/>

    <module name="TreeWalker">
        <module name="IndentationCheck">
<!--        <property name="tabWidth" value="4"/>-->
            <property name = "forceStrictCondition" value="true"/>
        </module>
    </module>
</module>public enum enum1 {

    ADD_Y(0, true, 1,
        m
                                      -> {
                        if (m.size() == 1) {
                            return m.getClass().getSimpleName().startsWith("add") || m.getClass().getSimpleName().startsWith("insert");
                        }
                return false;
        });

    private final Predicate<List<?>> detector;
    private final int level;
    private final boolean multi;
    private final int valueParameterIndex;

    enum1(int valueParameterIndex, boolean multi, int level, Predicate<List<?>> detector) {
        this.multi = multi;
        this.level = level;
        this.detector = detector;
        this.valueParameterIndex = valueParameterIndex;
    }
}
forceStrictCondition set to true
src % java -jar checkstyle-10.0-all.jar -c config.xml test.java                
Starting audit...
[WARN] /Users/shabana/Documents/OpenSource/CheckStyleTest/src/test.java:5:39: 'lambda' has incorrect indentation level 38, expected level should be 12. [Indentation]
[WARN] /Users/shabana/Documents/OpenSource/CheckStyleTest/src/test.java:6:25: 'if' has incorrect indentation level 24, expected level should be 12. [Indentation]
[WARN] /Users/shabana/Documents/OpenSource/CheckStyleTest/src/test.java:7:29: 'if' child has incorrect indentation level 28, expected level should be 16. [Indentation]
[WARN] /Users/shabana/Documents/OpenSource/CheckStyleTest/src/test.java:8:25: 'if rcurly' has incorrect indentation level 24, expected level should be 12. [Indentation]
[WARN] /Users/shabana/Documents/OpenSource/CheckStyleTest/src/test.java:9:17: 'block' child has incorrect indentation level 16, expected level should be 12. [Indentation]
Audit done.

forceStrictCondition set to false

src % java -jar checkstyle-10.0-all.jar -c config.xml test.java
Starting audit...
[WARN] /Users/shabana/Documents/OpenSource/CheckStyleTest/src/test.java:6:25: 'if' has incorrect indentation level 24, expected level should be one of the following: 12, 16, 42. [Indentation]
[WARN] /Users/shabana/Documents/OpenSource/CheckStyleTest/src/test.java:7:29: 'if' child has incorrect indentation level 28, expected level should be one of the following: 16, 20, 46. [Indentation]
[WARN] /Users/shabana/Documents/OpenSource/CheckStyleTest/src/test.java:8:25: 'if rcurly' has incorrect indentation level 24, expected level should be one of the following: 12, 16, 42. [Indentation]
Audit done.

Reply:
`forceStrictCondition` will apply to -> `{` and not if block

``` src % cat config.xml Test.java 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name = "Checker">
    <property name="charset" value="UTF-8"/>

    <property name="severity" value="warning"/>

    <property name="fileExtensions" value="java, properties, xml"/>

    <module name="TreeWalker">
        <module name="IndentationCheck">
<!--        <property name="tabWidth" value="4"/>-->
            <property name = "forceStrictCondition" value="false"/>
        </module>
    </module>
</module>public class Test {

    void test() {
            if (true) {
                return true;
            }
    }
}

src % java -jar checkstyle-10.0-all.jar -c config.xml Test.java
Starting audit...
[WARN] /Users/shabana/Documents/OpenSource/CheckStyleTest/src/Test.java:4:13: 'if' has incorrect indentation level 12, expected level should be 8. [Indentation]
[WARN] /Users/shabana/Documents/OpenSource/CheckStyleTest/src/Test.java:5:17: 'if' child has incorrect indentation level 16, expected level should be 12. [Indentation]
[WARN] /Users/shabana/Documents/OpenSource/CheckStyleTest/src/Test.java:6:13: 'if rcurly' has incorrect indentation level 12, expected level should be 8. [Indentation]
Audit done.


Otherwise, above also lvl of `if` should be accepted.

Reply:
Thanks for investigation, it looks like this is an existing bug (that is not entirely related), please see if there is an open issue for it, if not, let's open one.

Simple test case is like:
```
import java.util.stream.Stream;

public class Test {
    void method(Stream<String> ss) {
        ss.forEach(m -> {
                if (m.length() == 3) {
            }
        });
    }
}


```

Reply:
There is an open issue for `IndentationCheck` not behaving as expected when `forceStrictCondition` is set to true
https://github.com/checkstyle/checkstyle/issues/5448 https://github.com/checkstyle/checkstyle/issues/613

Reply:
Before creating an issue, I would like to know, indentation of `if` will be a case of `basicOffset`, since there is no line wrapping isn't it?,
```
ss.forEach(m -> {
                if (m.length() == 3) {
            
```

Here, there is line wrapping so any lvl is allowed and is also not reported as violation.
```
ss.forEach(m 
                           ->  {
                if (m.length() == 3) {
            
```


line 16-17 of first example https://checkstyle.sourceforge.io/config_misc.html#Indentation_Examples

Reply:
This is how I interpret the documentation:

```
ss.forEach(m -> { // basicOffset
                if (m.length() == 3) { // basicOffset
```

```
ss.forEach(m // basicOffset
                           ->  { // lineWrappingIndentation
                if (m.length() == 3) { // basicOffset

```
But, I find the documentation a bit confusing, since `forceStrictCondition` is false as default, it should have no effect on example given.


When `forceStrictCondition` is false, any indentation level greater than what is required should be OK.

I do not think that we need another issue, let's just add above example to https://github.com/checkstyle/checkstyle/issues/5448

Reply:
Resolved.